JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,15 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  let secs = seconds % 60;
+  let mins = Math.floor(seconds / 60) % 60;
+  let hours = Math.floor(seconds / 3600);
+
+  secs < 10 ? secs = `0${secs}` : secs;
+  mins < 10 ? mins = `0${mins}` : mins;
+  hours < 10 ? hours = `0${hours}` : mins;
```

Comment:
I would prefer have the variable definitions in this way, same logic but we had a better understanding of the logic operator

 secs = secs < 10 ? `0${secs}` : secs;
 mins = mins < 10 ? `0${mins}` : mins;
 hours = hours < 10 ? `0${hours}` : hours;

Reply:
That's definitely better, thanks for the feedback!